interfaces/api/service/IFrontEnd.h

```diff
@@ -32,6 +33,39 @@ namespace SolAR {
 namespace api {
 namespace service {
 
+/**
+ * @typedef DeviceType
+ * @brief <B>Define the types of all devices.</B>
+ */
+typedef enum {
+    OTHER_DEVICE = 0,
```

Comment:
UNKOWN_DEVICE to be consistent with DetectedObjectType ?

Reply:
Ca te va ça @Minipeps  ? Car ça implique une modif des proto côté proxy.

Reply:
So, no change for the moment, to avoid modifications on proxy side, but idea to keep for next version.